src/client/index.ts

```diff
@@ -25,6 +24,7 @@ interface NegotiationInterface {
     filters: {};
   };
   onChainKeys?: { [apiName: string]: string };
+  ipfsBaseUrl?: string
```

Comment:
Good idea :) Especially if we find people want to offload requests onto their servers.

Reply:
Yeah it make things a lot faster when using Moralis/Alchemy, which returns raw metadata that often includes ipfs links using public gateways. This is different from Opensea which returns a cached CDN URL.